<launch>

    <node pkg="convex_plane_decomposition_ros" type="save_elevation_map" name="save_elevation_map"
          output="screen" launch-prefix="">
        <param name="frequency" value="0.1"/>
        <param name="elevation_topic" value="/elevation_mapping/elevation_map_raw"/>
        <param name="height_layer" value="elevation"/>
        <param name="imageName" value="$(find convex_plane_decomposition_ros)/data/elevationMap"/>
    </node>

</launch>
